### PR TITLE
Docs: Slight clarification of what is durable in sharding.

### PR DESCRIPTION
**Pull request by helena**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/cluster-sharding.md [hunk 1]
```diff
@@ -295,18 +295,18 @@ This mode is enabled with configuration (enabled by default):
...
-The state of the coordinator is not durable, it's not stored to disk. When all nodes in
-the cluster have been stopped the state is lost and not needed any more.
+The state of the `ShardCoordinator` is replicated across the cluster but is not durable, not stored to disk.
+The `ShardCoordinator` state replication is handled by @ref:[Distributed Data](distributed-data.md) with `WriteMajority`/`ReadMajority` consistency.
+When all nodes in the cluster have been stopped, the state is no longer needed and dropped.
 
-The state of [Remembering Entities](#cluster-sharding-remembering) is also durable, i.e. it is stored to
-disk. The stored entities are started also after a complete cluster restart.
+The state of [Remembering Entities](#cluster-sharding-remembering) is durable and stored to
+disk. Thus stored entities are restarted after a complete cluster restart.
```
Comment (raboof) [inline akka-docs/src/main/paradox/cluster-sharding.md]: Perhaps in this Kubernetes world...
```suggestion
disk. This means remembered entities are restarted even after a complete (non-rolling) cluster restart when the disk is still available.
```

File: akka-docs/src/main/paradox/cluster-sharding.md [hunk 2]
```diff
@@ -295,18 +295,18 @@ This mode is enabled with configuration (enabled by default):
...
 
-Cluster Sharding is using its own Distributed Data `Replicator` per node role. In this way you can use a subset of
+Cluster Sharding is using its own Distributed Data `Replicator` per node. 
+If using roles with sharding, the `Replicator` is per node role which enables a subset of
```
Comment (raboof) [inline akka-docs/src/main/paradox/cluster-sharding.md]:
```suggestion
If using roles with sharding there is one `Replicator` per role, which enables a subset of
```

**Review by raboof — APPROVED** [review]
LGTM, some more wording ideas for your consideration

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**